Modules/StudyProgramme/classes/class.ilObjStudyProgramme.php

```diff
@@ -846,6 +849,9 @@ public function moveTo(ilObjStudyProgramme $a_new_parent) {
 	 * @throws ilException
 	 * @param  int 				$a_usr_id
 	 * @param  int | null		$a_assigning_usr_id	- defaults to global ilUser
+// !!!DIC refactoring-script warning.!!!
+// There is an isolated 'global' whithout any variable behind.
```

Comment:
@dkloepfer and @klees This comments seems to be unnecessary or you should have a look at this part of the code. But: I'm very interested in the script you use for the global -> DIC migration ;-) is it this script: https://github.com/ILIAS-eLearning/DeveloperTools/blob/master/global_to_dic/global_to_dic.php
thanks for the info!